cdk/test/functions/service-control-policies-root.test.ts

```diff
@@ -0,0 +1,252 @@
+
+const organizationClientMock = mockClient(OrganizationsClient);
+const rootAccountName = 'test-root-account';
+const rootAccountId = 'test-root-id';
+const logicalResourceId = 'superwerker - SCPRoot';
+const scpName = 'superwerker-root';
+const policyId = 'test-policy-id';
+const scpPolicy = '{"Version": "2012-10-17", "Statement": [/* policy document */]}';
+const updatedSCPPolicy = '{"Version": "2012-10-17", "Statement": [/* UPDATED policy document */]}';
+const description = 'superwerker - SCPRoot';
+
+describe('service control policies', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    organizationClientMock.reset();
+    process.env.AWS_REGION = 'us-east-1';
+  });
+
+  afterEach(() => {
+    delete process.env.AWS_REGION;
+  });
+
+  it('create service control policy', async () => {
+    organizationClientMock.on(ListRootsCommand).resolves({
+      Roots: [
+        {
+          Id: rootAccountId,
+          Name: rootAccountName,
+        },
+      ],
+    });
+
+    organizationClientMock.on(ListPoliciesCommand).resolves({
+      Policies: [
+        {
+          Name: scpName,
+          Id: logicalResourceId,
+          Description: description,
+        },
+      ],
+    });
+
+    organizationClientMock.on(CreatePolicyCommand).resolves({
+      Policy: {
+        PolicySummary: {
+          Name: scpName,
+          Id: policyId,
+        },
+      },
+    });
+
+    organizationClientMock.on(AttachPolicyCommand).resolves({});
+
+    const response = await handler(
+      {
+        RequestType: 'Create',
+        ResourceProperties: {
+          Type: PolicyType.SERVICE_CONTROL_POLICY,
+          Description: logicalResourceId,
+          Name: scpName,
+          Content: scpPolicy,
+        },
+      } as unknown as CloudFormationCustomResourceCreateEvent,
+      {} as Context,
+    );
+
+    expect(response).toMatchObject({ SUCCESS: 'SCPs have been successfully created for Root account' });
+  });
+
+  it('update service control policy', async () => {
+    organizationClientMock.on(ListPoliciesCommand).resolves({
+      Policies: [
+        {
+          Name: scpName,
+          Id: logicalResourceId,
+          Description: description,
+        },
+      ],
+    });
+
+    organizationClientMock.on(UpdatePolicyCommand).resolves({
+      Policy: {
+        Content: '{"Version": "2012-10-17", "Statement": [/* UPDATED policy document */]}',
+        PolicySummary: {
+          Id: policyId,
+          Name: scpName,
+        },
+      },
+    });
+
+    const response = await handler(
+      {
+        RequestType: 'Update',
+        ResourceProperties: {
+          Type: PolicyType.SERVICE_CONTROL_POLICY,
+          Description: logicalResourceId,
+          scpName: scpName,
+          Content: updatedSCPPolicy,
+        },
+      } as unknown as CloudFormationCustomResourceCreateEvent,
+      {} as Context,
+    );
+
+    expect(response.Policy.Content).toEqual(updatedSCPPolicy);
```

Comment:
please check both: that the policy is the new one as well as the other properties

Reply:
Updated the expect statements.